### PR TITLE
change ucp_id to id and update institution ids

### PR DESCRIPTION
**Pull request by Craiting**
Updates ucp_ids following this MR https://github.com/Universal-Connect-Project/ucp-hosted-app/pull/56

### CODE REVIEW

File: apps/server/src/connect/connectApi.ts [hunk 1]
```diff
@@ -25,24 +25,25 @@ function mapResolvedInstitution(ins: Institution) {
...
-  ins: CachedInstitution
+  ins: CachedInstitution,
 ): InstitutionSearchResponseItem {
-  const supportsOauth = ins?.mx?.supports_oauth || ins?.sophtron?.supports_oauth
+  const supportsOauth =
+    ins?.mx?.supports_oauth || ins?.sophtron?.supports_oauth;
   // || ins.finicity.supports_oauth || ins.akoya.supports_oauth
   return {
-    guid: ins.ucp_id,
+    guid: ins.id,
```
Comment (tphalp) [inline apps/server/src/connect/connectApi.ts:41]: A lot of this is Prettier, so I'm putting comments to show where the ID changes are.

File: apps/server/src/services/institutionSyncer.test.ts [hunk 2]
```diff
@@ -1,172 +1,172 @@
...
-    const oldInstitutionUcpId = 'UCP-old1'
+  it("should call update methods if response is 200", async () => {
+    const infoLogSpy = jest.spyOn(logger, "info");
+    const oldInstitutionUcpId = "UCP-old1";
 
     await overwriteSet(INSTITUTION_CURRENT_LIST_IDS, [
       oldInstitutionUcpId,
-      elasticSearchInstitutionData.ucp_id
-    ])
+      elasticSearchInstitutionData.id,
```
Comment (tphalp) [inline apps/server/src/services/institutionSyncer.test.ts:77]: ID Change

File: apps/server/src/shared/contract.ts [hunk 3]
```diff
@@ -61,207 +61,207 @@ export enum ConnectionStatus {
...
-  url: string
-  ucp_id: string
-  is_test_bank: boolean | false
-  routing_numbers: string[]
-  [propName: string]: any
+  name: string;
+  keywords: string | null;
+  logo: string;
+  url: string;
+  id: string;
```
Comment (tphalp) [inline apps/server/src/shared/contract.ts:154]: ID change

File: apps/server/src/test/testData/institution.ts [hunk 4]
```diff
@@ -44,115 +44,115 @@ export const finicityInsitutionData = {
...
-  name: 'MX Bank (Oauth)',
-  keywords: 'Gringotts, MX Bank, oauth',
-  logo: 'https://s3.amazonaws.com/MD_Assets/Ipad%20Logos/100x100/INS-3aeb38da-26e4-3818-e0fa-673315ab7754_100x100.png',
-  url: 'https://mx.com',
-  ucp_id: 'UCP-da107e6d0da7779',
+  name: "MX Bank (Oauth)",
+  keywords: "Gringotts, MX Bank, oauth",
+  logo: "https://s3.amazonaws.com/MD_Assets/Ipad%20Logos/100x100/INS-3aeb38da-26e4-3818-e0fa-673315ab7754_100x100.png",
+  url: "https://mx.com",
+  id: "048b9a67-53ac-453c-9bff-baf418282311",
```
Comment (tphalp) [inline apps/server/src/test/testData/institution.ts:93]: ID change

**Review by tphalp — APPROVED** [review]

**Review by wesrisenmay-mx — APPROVED** [review]

**Pull request merged**